### PR TITLE
Anastasiia Bond

### PR DESCRIPTION
**Pull request by alianInGitHub**

### CODE REVIEW

File: TestEcoMap/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+chromedriver
+.idea
```
Comment (timbortnik) [inline TestEcoMap/.gitignore:2]: Can we have an EOL here?

**Review by timbortnik — COMMENTED** [review]
TestEcoMap$ nosetests
.
----------------------------------------------------------------------
Ran 1 test in 8.859s

OK
(yey)

File: TestEcoMap/TestBox/Test.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+from pages.PagesURL import PagesURL
+
+class MainPageTest(Test):
+    def test_open_main_page(self):
+        main_page = self.page_factory.get_page_object(self.driver, "main", PagesURL.MAIN)
+        main_page.open()
+        main_page.click_login()
+        login_page = self.page_factory.get_page_object(self.driver, "login", PagesURL.LOGIN)
+        login_page.enter_credentials(ADMIN.LOGIN, ADMIN.LOGIN)
+
```
Comment (timbortnik) [inline TestEcoMap/TestBox/Test.py:29]: Shall we assert that login actually happened?

File: TestEcoMap/TestBox/Test.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+if __name__== '__main__':
+     unittest.main()
```
Comment (timbortnik) [inline TestEcoMap/TestBox/Test.py:32]: This did not work for me
$ python TestBox/Test.py
Traceback (most recent call last):
  File "TestBox/Test.py", line 2, in <module>
    from pages.PageFactory import PageFactory
ImportError: No module named pages.PageFactory

Do we ever need to cal this out of unittest?

**Comment by timbortnik** [comment]
Would be great to fix style issues
image

**Review by timbortnik — CHANGES_REQUESTED** [review]
Great PR, it actually works!
Would be nice to address minor issues.

**Comment by nklymenok** [comment]
It would be better to read User`s info from the file.
For me not easy to understand this realisation, need more time  :)

**Review by timbortnik — APPROVED** [review]

**Pull request closed without merging**